Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/compute_normal.h

```diff
@@ -40,19 +40,38 @@ namespace Polygon_mesh_processing{
 
 namespace internal {
 
-  template<typename Point>
-  typename CGAL::Kernel_traits<Point>::Kernel::Vector_3
-  triangle_normal(const Point& p0, const Point& p1, const Point& p2)
+  template <class GT>
+  void normalize(typename GT::Vector_3& v, const GT& traits)
   {
-    return CGAL::cross_product(p2 - p1, p0 - p1);
+    double norm = CGAL::approximate_sqrt(
+        traits.compute_squared_length_3_object()(v));
+    v = traits.construct_divided_vector_3_object()(v, norm);
+  }
+
+  template<typename Point
+         , typename GT = typename CGAL::Kernel_traits<Point>::Kernel>
+  typename GT::Vector_3
```

Comment:
... for function templates.

For class templates that is allowed in C++98 as well.